lib/fizzy/capi.cpp

```diff
@@ -168,6 +168,27 @@ inline std::vector<fizzy::ImportedFunction> unwrap(
     return imported_functions;
 }
 
+inline fizzy::table_elements* unwrap(FizzyTable* table) noexcept
+{
+    return reinterpret_cast<fizzy::table_elements*>(table);
+}
+
+inline fizzy::Limits unwrap(const FizzyLimits& limits) noexcept
+{
+    return {limits.min, limits.has_max ? std::make_optional(limits.max) : std::nullopt};
+}
+
+inline fizzy::ExternalTable unwrap(const FizzyExternalTable& external_table) noexcept
+{
+    return {unwrap(external_table.table), unwrap(external_table.limits)};
+}
+
+inline std::vector<fizzy::ExternalTable> unwrap(const FizzyExternalTable* external_table)
+{
+    return external_table ? std::vector<fizzy::ExternalTable>{unwrap(*external_table)} :
+                            std::vector<fizzy::ExternalTable>{};
```

Comment:
Cannot just `{}` work here?

Reply:
No, `error: initializer list cannot be used on the right hand side of operator '?'`